<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1560</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
.QSpinBox
{
	color:white;	
	padding: 4px;
}


.QLabel
{
	border: 1px solid yellow;
}
                                 


QWidget
{
	background-color: #232430;
	color: #000000;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: #232430;
	color: #c1c1c1;
	border-color: #000000;
}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #ff9c2b;
	color: #000000;
	font-weight: bold;
	border-style: solid;
	border-color: #000000;
	padding: 6px;

}


QPushButton::hover
{
	background-color: #ffaf5d;

}


QPushButton::pressed
{
	background-color: #dd872f;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: #ff9c2b;
	color: #000000;
	font-weight: bold;
	border-style: solid;
	border-color: #000000;
	padding: 6px;

}


QToolButton::hover
{
	background-color: #ffaf5d;

}


QToolButton::pressed
{
	background-color: #dd872f;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #38394e;
	color: #c1c1c1;
	border-style: solid;
	border-width: 1px;
	border-color: #4a4c68;

}


/*-----QTableView-----*/
QTableView, 
QHeaderView, 
QTableView::item 
{
	background-color: #232430;
	color: #c1c1c1;

}


QTableView::item:selected 
{ 
    background-color: #41424e;
    color: #c1c1c1;

}


QHeaderView::section:horizontal 
{
    background-color: #232430;
	border: 1px solid #37384d;
	padding: 5px;

}


QTableView::indicator{
	background-color: #1d1d28;
	border: 1px solid #37384d;

}


QTableView::indicator:checked{
	image:url(&quot;./ressources/check.png&quot;); /*To replace*/
	background-color: #1d1d28;

}

/*-----QTabWidget-----*/
QTabWidget::pane 
{ 
    border: none;

}


QTabWidget::tab-bar 
{
    left: 5px; 

}


QTabBar::tab 
{
    color: #c1c1c1;
    min-width: 1px;
	padding-left: 25px;
	margin-left:-22px;
    height: 28px;
	border: none;

}


QTabBar::tab:selected 
{
    color: #c1c1c1;
	font-weight: bold;
    height: 28px;

}


QTabBar::tab:!first 
{
    margin-left: -20px;

}


QTabBar::tab:hover 
{
    color: #DDD;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal 
{
    background-color: transparent;
    height: 8px;
    margin: 0px;
    padding: 0px;

}


QScrollBar::handle:horizontal 
{
    border: none;
	min-width: 100px;
    background-color: #56576c;

}


QScrollBar::add-line:horizontal, 
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal, 
QScrollBar::sub-page:horizontal 
{
    width: 0px;
    background-color: transparent;

}


QScrollBar:vertical 
{
    background-color: transparent;
    width: 8px;
    margin: 0;

}


QScrollBar::handle:vertical 
{
    border: none;
	min-height: 100px;
    background-color: #56576c;

}


QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical, 
QScrollBar::sub-page:vertical 
{
    height: 0px;
    background-color: transparent;

}

QCalendarWidget
{
	 border: 5px solid yellow;
}
</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>1551</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Choose Picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>561</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Original Image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_20">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>620</y>
      <width>821</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_21">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Â© M. S. Lovushko, Donskoy State Technical University, 2023</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Return to original</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Change Gaussian Noize</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sobel Result</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_15">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Viewlet Results</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>250</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>430</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_16">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>70</y>
      <width>561</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>81</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>510</y>
      <width>561</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <row>
        <property name="text">
         <string>DOG</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>WAVE</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>MATH</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>WAVE_THRESHOLD</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>DOG_THRESHOLD</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>MATH_THRESHOLD</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>SOBEL</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Mean Square Deviation</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>MSD(%)</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
